pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -318,6 +320,10 @@ func checkCertValidity(certBytes, keyBytes []byte, ipAddresses []string, nodeUID
 	// Check that the cert is valid for the provided ip addresses
 	errs := []error{}
 	for _, ipAddress := range ipAddresses {
+		if net.IsIPv6String(ipAddress) {
+			klog.Warningf("IPv6 address %q was detected for node %q, it will not be added to the certificate", ipAddress, nodeUID)
```

Comment:
this is bad in a verify func. It's not clear how often this is called, potentially very often in the sync loop. I also don't see why this should be a warning in the first place.